WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -72,8 +76,8 @@ public partial class ChangeAvoidanceSuggestionViewModel : SuggestionViewModel
 		return normalized;
 	}
 
-	public static async Task<IEnumerable<ChangeAvoidanceSuggestionViewModel>> GenerateSuggestionsAsync(
-		TransactionInfo transactionInfo, BitcoinAddress destination, Wallet wallet, BuildTransactionResult requestedTransaction)
+	public static async IAsyncEnumerable<ChangeAvoidanceSuggestionViewModel> GenerateSuggestionsAsync(
+		TransactionInfo transactionInfo, BitcoinAddress destination, Wallet wallet, BuildTransactionResult requestedTransaction, [EnumeratorCancellation] CancellationToken cancellationToken)
```

Comment:
This `CancellationToken` became the enumartion's cancellation token. 
Is it OK to pass the same token to the BnB algo? Or we should use a different CTS for the algo?

WDYT? @kiminuo

Reply:
yeah, i think it should work well.